h5grove/utils.py

```diff
@@ -16,6 +23,10 @@ def get_entity_from_file(
         return h5file[path]
 
     link = h5file.get(path, getlink=True)
+
+    if link is None:
+        raise PathError(f"{path} is not a valid path in {basename(h5file.filename)}")
```

Comment:
I made it work by raising a `PathError` that is caught and dealt with appropriately in `tornado/flaskutils`

I use a custom error because I want the behaviour to be specific to this error and not catch unrelated `KeyError` for example